src/dialogs/prompt/index.js

```diff
@@ -27,6 +65,7 @@ module.exports = async function showPrompt (options) {
     show: false,
     width: 350,
     height: 330,
+    useContentSize: true,
```

Comment:
Yes. `useContentSize`, as the linked docs says, means:

> The width and height would be used as web page's size, which means the actual window's size will include window frame's size and be slightly larger

`useContentSize` is used in order to ensure that the content part of the window is actually the height we set. Otherwise, the height we set will also include the frame and menu size's, which we can't estimate because they vary cross-platform.